lib/validate_partitions.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+package validate_partitions;
```

Comment:
we can pick a name that resemble more to a library name, like `partitioning_validator_utils` or `partitioning_validator`, but in general a noun not a verb.